Examples/Tests/PML/inputs_3d

```diff
@@ -16,7 +16,8 @@ boundary.field_hi = pml pml pml
 # Numerical parameters
 warpx.cfl = 1.0
 warpx.do_nodal = 0
-warpx.do_dive_cleaning = 0
+warpx.do_dive_cleaning = 1
+warpx.do_divb_cleaning = 1
```

Comment:
Note: I activated div(E) and div(B) cleaning also in the regular domain for this CI test. Consequently, I also set `psatd.update_with_rho = 1` below. The relatively large changes in the benchmark are in most part due to switching from `psatd.update_with_rho = 0` to `psatd.update_with_rho = 1`, and not to the use of div(E) and div(B) cleaning in the regular domain.